tests/model_hub_tests/torch_tests/test_hf_transformers.py

```diff
@@ -286,8 +347,13 @@ def forward(self, flattened_patches, attention_mask):
             except:
                 pass
         if model is None:
-            from transformers import AutoModel
-            model = AutoModel.from_pretrained(name, torchscript=True)
+            from transformers import AutoModel, AutoConfig
+            config = AutoConfig.from_pretrained(name)
```

Comment:
Could you patch config in the beginning?

Reply:
moved